README.md

```diff
@@ -51,3 +51,25 @@ so that you can use the OpenAI API SDKs with keyless (Entra) authentication. By
     ```
 
     This will use the OpenAI API SDK to make a request to the OpenAI API and print the response.
+
+
+## Creating a new azd enviroment 
+
+1. If you'd like to create a new Python project, you can create a new azd environment for it.
+Create a new environment and set it as the default by running:
+
+    ```shell
+    azd env new <environment>
+    ```
+
```

Comment:
Can you add a step like "Then run the commands above in #provisioning to provision the new Azure OpenAI resource and update your `.env` file." ?